app/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,9 @@
 <resources>
+    <string name="app_name">FInalProject</string>
+
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```

Comment:
This is a default String, that you should not use.... if you injected it now, it will be hard to take it out later